python/src/IteratedQuadrature_doc.i.in

```diff
@@ -37,59 +43,16 @@ Create an iterated quadrature algorithm:
 >>> l = [ot.SymbolicFunction(['x'], [' 2+cos(x)'])]
 >>> u = [ot.SymbolicFunction(['x'], ['-2-cos(x)'])]
 
-Draw the graph of the integrand and the bounds:
-
->>> g = ot.Graph('Integration nodes', 'x', 'y', True, 'upper right')
->>> g.add(f.draw([a,a],[b,b]))
->>> curve = l[0].draw(a, b).getDrawable(0)
->>> curve.setLineWidth(2)
->>> curve.setColor('red')
->>> g.add(curve)
->>> curve = u[0].draw(a, b).getDrawable(0)
->>> curve.setLineWidth(2)
->>> curve.setColor('red')
->>> g.add(curve)
-
 Evaluate the integral with high precision:
 
 >>> Iref = ot.IteratedQuadrature(ot.GaussKronrod(100000, 1e-13, ot.GaussKronrodRule(ot.GaussKronrodRule.G11K23))).integrate(f, a, b, l, u)
 
-Evaluate the integral with the default GaussKronrod algorithm, and get
-evaluation points:
-
->>> f = ot.MemoizeFunction(f)
->>> I1 = ot.IteratedQuadrature(ot.GaussKronrod()).integrate(f, a, b, l, u)
->>> sample1 = f.getInputHistory()
->>> print(I1)
-[-25.132...]
->>> n_evals = sample1.getSize()
->>> print(n_evals)
-2116
->>> err = abs(100.0*(1.0-I1[0]/Iref[0]))
->>> print(err)
-2.2...e-14
->>> cloud = ot.Cloud(sample1)
->>> cloud.setPointStyle('fcircle')
->>> cloud.setColor('green')
->>> g.add(cloud)
->>> f.clearHistory()
-
 Evaluate the integral with the default IteratedQuadrature algorithm:
 
->>> I2 = ot.IteratedQuadrature().integrate(f, a, b, l, u)
->>> sample2 = f.getInputHistory()
->>> print(I2)
-[-25.132...]
->>> n_evals = sample2.getSize()
->>> print(n_evals)
-5236
->>> err = abs(100.0*(1.0-I2[0]/Iref[0]))
+>>> Idefault = ot.IteratedQuadrature().integrate(f, a, b, l, u)
+>>> err = abs(100.0*(1.0-Idefault[0]/Iref[0]))
```

Comment:
This code is surprising.
1. Please use blank spaces around arithmetic operators.
2. It seems to me that this code computes the relative error with respect to the high accuracy reference value computed earlier in the script. If this is true, then I suggest to remove the multiplication by 100, because it is not relevant in my opinion.
```suggestion
>>> relative_error = abs(1.0 - Idefault[0] / Iref[0])
```